include/teb_local_planner/teb_config.h

```diff
@@ -104,6 +104,7 @@ class TebConfig
     bool is_footprint_dynamic; //<! If true, updated the footprint before checking trajectory feasibility
     bool use_proportional_saturation; //<! If true, reduce all twists components (linear x and y, and angular z) proportionally if any exceed its corresponding bounds, instead of saturating each one individually
     double transform_tolerance = 0.5; //<! Tolerance when querying the TF Tree for a transformation (seconds)
+    double min_vel_x;
```

Comment:
move together with the other velocities (e.g. after max_vel_x), as on .cfg and add the same comment here